js/event-domcontentloaded/demos/basic/index.html

```diff
@@ -15,7 +15,11 @@
     }
 
```

Comment:
В демке скроллы есть, их бы покрасить

```suggestion

    html {
      color-scheme: dark;
    }

```